src/views/profile.view.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Cronbeat\Views;
+
+class ProfileView extends BaseView {
+    protected ?string $error = null;
+    protected ?string $success = null;
+    protected string $username = '';
+    protected string $name = '';
+    protected string $email = '';
+
+    public function __construct() {
+        $this->setTitle('Your Profile');
+        // Reuse dashboard container styling for profile while keeping a specific hook
+        $this->setContainerClass('dashboard-container profile-container');
```

Comment:
@jetbrains-junie this is hardly maintainable. go with `view-container profile-container` and move the css from `dashboard-container` to `view-container`. In dashboard, also use `view-container`.

this way we have basic styling with the option to identify each particular container later and add custom styling.